<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(99, 143, 172);
}
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>30</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:3px solid#63bdc5;
border-radius:200px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>8000000</width>
      <height>8000000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>                显示视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetinfo">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
     </font>
    </property>
    <property name="text">
     <string>信息录入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_timer">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{color: rgb(255, 255, 255);
background-color: rgb(135, 209, 216);
border:4px solid#87d1d8;
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string>    showtmie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:5px solid#63bdc5;
border-radius:30px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:2px dotted#63bdc5;
border-radius:120px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>295</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOpenCam">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>隶书</family>
        </font>
       </property>
       <property name="text">
        <string>打开摄像头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStaring">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>隶书</family>
        </font>
       </property>
       <property name="text">
        <string>识别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCloseCam">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>隶书</family>
        </font>
       </property>
       <property name="text">
        <string>关闭摄像头</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblImage_2">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>30</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99, 143, 172);
border-image: url(:/open.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;学号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="namelabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>190</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numberlabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>190</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trust">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;置信度：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>lblImage_2</zorder>
   <zorder>lblImage</zorder>
   <zorder>imageLabel</zorder>
   <zorder>btnGetinfo</zorder>
   <zorder>label_timer</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>namelabel</zorder>
   <zorder>numberlabel</zorder>
   <zorder>trust</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
